journal-letters/original/16-chapter-15.adoc

```diff
@@ -145,6 +150,7 @@ nor make shipwreck of our faith; which is in the Lord Jehovah, who is over all,
 blessed forevermore!`"
 
 William Caton.
+--
```

Comment:
Oops, we missed the signed-section-signature above his name on line 152 (which you did get in the modernized version).